docs/docs/features/proposals.md

```diff
@@ -77,6 +77,13 @@ Minimal example:
 }
 ```
 
+:::warning
+Before the introduction of Partial Set Security, consumer chains typically included a "soft opt-out mechanism"
+which allows the bottom N% of the provider's validators to not validate the consumer chain, without being jailed for downtime on the provider.
+After the introduction of Partial Set Security, the use of the soft opt-out mechanism is discouraged, and consumer chains are
+encouraged to use the topN parameter to allow validators with little power to opt-out.
```

Comment:
> [...] to allow validators with little power to opt-out.
They do not technically opt out. They are not just force to opt in. So, maybe rephrasing it to:
> [...] to not force validators with litter power to validate the chain.

Reply:
True, fixed!